src/tests/ftest/launch.py

```diff
@@ -428,6 +433,14 @@ def get_logs_dir(self):
 
         """
         default_base_dir = os.path.join("~", "avocado", "job-results")
+        logger.info("default_base_dir = %s", default_base_dir)
+        datadir_paths_logs_dir = self.get_setting("datadir.paths", "logs_dir", default_base_dir)
+        logger.info("datadir.paths = %s ", datadir_paths_logs_dir)
+        time.sleep(2)
+        datadir_paths_logs_dir = self.get_setting("datadir.paths", "logs_dir", default_base_dir)
+        logger.info("datadir.paths = %s ", datadir_paths_logs_dir)
```

Comment:
If we want to keep these log messages, they should not be info, but debug to avoid cluttering the console:
```suggestion
        logger.debug("default_base_dir = %s", default_base_dir)
        datadir_paths_logs_dir = self.get_setting("datadir.paths", "logs_dir", default_base_dir)
        logger.debug("datadir.paths = %s ", datadir_paths_logs_dir)
        time.sleep(2)
        datadir_paths_logs_dir = self.get_setting("datadir.paths", "logs_dir", default_base_dir)
        logger.debug("datadir.paths = %s ", datadir_paths_logs_dir)
```

Reply:
This is all just part of the process of getting a new distro to run in CI.  Lots of debug code that will ultimately be removed.